scripts/enable_automatic_updates.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+apt-get install unattended-upgrades curl
+
+echo '
+APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Unattended-Upgrade "1";
+' >/etc/apt/apt.conf.d/20auto-upgrades
+
+sed -i.bak 's|//Unattended-Upgrade::Remove-Unused-Dependencies "false"|Unattended-Upgrade::Remove-Unused-Dependencies "true"|' /etc/apt/apt.conf.d/50unattended-upgrades
+
+sed -i.bak 's|//Unattended-Upgrade::Automatic-Reboot "false"|Unattended-Upgrade::Automatic-Reboot "false"|' /etc/apt/apt.conf.d/50unattended-upgrades
+
+rm /etc/apt/apt.conf.d/50unattended-upgrades.bak
+
+echo '
+APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Download-Upgradeable-Packages "1";
+APT::Periodic::AutocleanInterval "1";
+APT::Periodic::Unattended-Upgrade "1";
+APT::Periodic::Verbose 2;
+APT::Periodic::RandomSleep 1;
+'> /etc/apt/apt.conf.d/10periodic
```

Comment:
Is there a good place to link to docs for what each value does? They're pretty self explanatory, but it might still be good.

Reply:
I've pre-fixed a comment linking to the ubuntu docs page.